libs/common/src/lib/security/authenticatedRouteMiddleware.ts

```diff
@@ -6,7 +8,7 @@ import { IJwtTokenData } from '@spotlyt-backend/data/interfaces';
 
 
 export const Authenticate = createParamDecorator(
-    (roles: unknown, ctx: ExecutionContext) => {
+    (args: unknown, ctx: ExecutionContext) => {
```

Comment:
Why args are `unknown`? Can we get the types for them

Reply:
its not required as of now, cause we are doing authorization via CASL